src/textual/driver.py

```diff
@@ -117,6 +114,9 @@ def write(self, data: str) -> None:
             data: Raw data.
         """
 
+    def flush(self) -> None:
```

Comment:
Missing decorator?

Reply:
No, this one doesn't need to be abstract. Default is to to nothing...